protocol-units/da/m1/light-node/src/main.rs

```diff
@@ -13,7 +14,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 	let dot_movement = dot_movement::DotMovement::try_from_env()?;
 	let config_path = dot_movement.get_config_json_path();
 	let config_file = tokio::fs::File::open(config_path).await?;
-	let manager = Manager::<LightNodeV1>::new(config_file).await?;
+	// todo: consider whether LightNode implementation should encapsulate signing type
```

Comment:
should probably be generic if target is multiple possible DAs, correct?

Reply:
At the level of running in `main.rs`, this is where we have to decide. So, it's generic in the lib, but the bin shan't be.